workers/unity/Packages/io.improbable.gdk.tools/GenerateCode.cs

```diff
@@ -211,13 +255,44 @@ private static void ProcessStdOut(string output)
                         break;
                 }
             }
-            else
+        }
+
+        private static void ProcessCodegenOutput(string output)
+        {
+            try
+            {
+                var log = CodegenLog.FromRaw(output);
+                latestCodegenLogs.Append(log);
```

Comment:
If we only need this list for counting the warns/errors we should instead keep a `Dictionary<CodegenLogLevel, int>` counter instead of storing all messages and then iterate over them to count

Reply:
Yeah I agree

I originally wanted it as a list so we can use it in the Future Codegen Output Window™️, but it probably won't be happening anytime soon so 🤷‍♂